containers/hugging-face-inference/README.md

```diff
@@ -0,0 +1,33 @@
+# Hugging Face Models
+
+### Deploy models in Serverless Containers
+
+- Export these variables:
+
+```bash
+export SCW_ACCESS_KEY="access-key" SCW_SECRET_KEY="secret-key" SCW_PROJECT_ID="project-id" REGION="fr-par"
```

Comment:
**suggestion(tfvars):** make these Terraform variables and give region a default of `fr-par`.